cub/cub/device/device_segmented_reduce.cuh

```diff
@@ -99,47 +164,11 @@ struct DeviceSegmentedReduce
   //!
   //! The code snippet below illustrates a custom min-reduction of a device vector of ``int`` data elements.
   //!
-  //! .. code-block:: c++
-  //!
-  //!    #include <cub/cub.cuh>
-  //!    // or equivalently <cub/device/device_radix_sort.cuh>
-  //!
-  //!    // CustomMin functor
-  //!    struct CustomMin
-  //!    {
-  //!        template <typename T>
-  //!        CUB_RUNTIME_FUNCTION __forceinline__
-  //!        T operator()(const T &a, const T &b) const {
-  //!            return (b < a) ? b : a;
-  //!        }
-  //!    };
```

Comment:
question: we used to have `CustomMin` example in the doc, but now its definition is out of the literal include scope. Do you think it's useful to have another literal include to provide user with an example of function object?